src/components/Button/Button.js

```diff
@@ -66,9 +63,6 @@ Button.defaultProps = {
 Button.propTypes = {
   ...propTypes,
 
-  /** @type {boolean} Whether or not the button is disabled. */
-  disabled: PropTypes.bool,
-
   /** @type {boolean} Whether or not the button is in a loading state. While loading, the button is disabled & icons provided via the `renderIcon` prop will not be shown. */
```

Comment:
This prop description will need to be updated 👍

Reply:
What do you mean here?